specs/simple-serialize.md

```diff
@@ -89,7 +89,7 @@ An SSZ object is called empty (and thus, `is_empty(object)` returns true) if it
 
 ### Illegal types
 
-Empty vector types (i.e. `[subtype, 0]` for some `subtype`) are not legal. The `null` type is only legal as the first type in a union subtype (i.e. with type index zero).
+The empty `Vector[subtype, N]` and `Bitvector[N]` types, where `N == 0` are not legal. The `null` type is only legal as the first type in a union subtype (i.e. with type index zero).
```

Comment:
```suggestion
- Empty vector types (Vector[T, 0], Vector[byte, 0], bitvector)
- Containers with no fields
- The `null` type is only legal as the first type in a union subtype (i.e. with type index zero).
```

Reply:
This is the other end of the proposal, keep it stupid simple:
- no empty vectors, we do not even check if the element type is variable size (which would result in allowable `Vector[T, 0]`). Instead, just no empty vectors
- no empty containers. (if they are fixed-size, they result in the same 0-length non-offset element type problem)

Reply:
Yeah, I'm inclined to this (disable empty vector and container). 
Modified your suggestion in 62138fe.